cmd/ipfs/pinmfs.go

```diff
@@ -227,6 +225,10 @@ func pinMFS(
 			pinTime = ps.GetCreated().UTC()
 			break
 		}
+		//Run a goroutine to monitor status of queued pins
+		if ps.GetPin().GetCid() == cid && ps.GetStatus() == pinclient.StatusQueued {
+			checkPinStatus(ctx, node, svcName, svcConfig, ps, repinInterval)
+		}
```

Comment:
I do not think we want to start a new goroutine for each pin. I think it would be better to communicate pins to watch to a single goroutine.